src/Control/Session.php

```diff
@@ -329,10 +329,27 @@ public function start(HTTPRequest $request)
 
                 session_start();
 
+                $sameSite = null;
+                $sessionParams = session_get_cookie_params();
```

Comment:
Does this work? It's calling `session_set_cookie_params` above without `samesite`.
From my testing, the above call removes the `samesite`, and the below `Cookie::set` will only get called if you also have a non-zero `$timeout` configured.

Reply:
@madmatt ?